cea/interfaces/dashboard/plots/templates/manage.html

```diff
@@ -69,7 +73,7 @@ <h2 class="pull-left">Manage Dashboards</h2>
   {{ super() }}
   <script>
     function openSettings(dashboard_index, func) {
-        $.get('./' + dashboard_index + '/' + func, {}, function(html) {
+        $.get('./' + func + '/' + dashboard_index, {}, function(html) {
```

Comment:
yeah, i still think this needs to be refactored (see above) -  https://refactoring.com/catalog/removeFlagArgument.html